tests/test_datasets.py

```diff
@@ -173,7 +173,7 @@ def test_download_failure():
                 *DS_MARKS[ds],
             ],
         )
-        for ds in set(sc.datasets.__all__) - DS_DYNAMIC
+        for ds in sorted(set(sc.datasets.__all__) - DS_DYNAMIC)
```

Comment:
This fixes `pytest -n auto`. Pytest complained locally that different jobs collected “different tests” which were just differently sorted.